linera-indexer/lib/src/runner.rs

```diff
@@ -49,13 +47,8 @@ impl<DB, Config> Runner<DB, Config>
 where
     Self: Send,
     Config: Clone + std::fmt::Debug + Send + Sync + clap::Parser + clap::Args,
-    DB: AdminKeyValueStore<Error = <DB as KeyValueStore>::Error>
-        + KeyValueStore
-        + Clone
-        + Send
-        + Sync
-        + 'static,
-    <DB as KeyValueStore>::Error: From<bcs::Error>
+    DB: KeyValueStore + Clone + Send + Sync + 'static,
```

Comment:
This should probably be `S` instead of `DB`, for consistency. (Also in other files.)

Reply:
Ok.